hammr/commands/image/image.py

```diff
@@ -217,52 +217,51 @@ def arg_deploy(self):
     def do_deploy(self, args):
         try:
             # add arguments
-            doParser = self.arg_deploy()
-            doArgs = doParser.parse_args(shlex.split(args))
+            do_parser = self.arg_deploy()
+            do_args = do_parser.parse_args(shlex.split(args))
 
             # if the help command is called, parse_args returns None object
-            if not doArgs:
+            if not do_args:
                 return 2
 
-            pimage = self.get_pimage_from_id(doArgs.pid)
-            if pimage == 2:
-                return 2
+            publish_image = self.get_publish_image_from_publish_id(do_args.pid)
 
-            target_platform = ""
-            if pimage.targetFormat:
-                target_platform = pimage.targetFormat.name
+            if not self.is_publish_image_ready_to_deploy(publish_image):
+                raise ValueError("Published image with id '" + do_args.pid + " is not ready to be deployed")
 
-            if not self.is_pimage_ready_to_deploy(pimage):
-                printer.out("Published image with name '" + pimage.name + " cannot be deployed", printer.ERROR)
-                return 2
+            deploy_file = generics_utils.get_file(do_args.file)
+            if deploy_file is None:
+                raise TypeError("Deploy file cannot be None")
```

Comment:
The error message is not understandable for a user: maybe something like "Deploy file not found" ?